static/app/components/events/contexts/utils.tsx

```diff
@@ -63,6 +65,39 @@ export function getRelativeTimeFromEventDateCreated(
   );
 }
 
+// Typescript doesn't have types for DisplayNames yet and that's why the type assertion "any" is needed below.
+// There is currently an open PR that intends to introduce the types https://github.com/microsoft/TypeScript/pull/44022
+export function getFullLanguageDescription(locale: string) {
+  const sentryAppLanguageCode = ConfigStore.get('languageCode');
+
+  const [languageAbbreviation, countryAbbreviation] = locale.includes('_')
+    ? locale.split('_')
+    : locale.split('-');
+
+  try {
+    const languageNames = new (Intl as any).DisplayNames(sentryAppLanguageCode, {
```

Comment:
Might be good to comment here about the `as any` cast: https://github.com/microsoft/TypeScript/issues/41338